modules/tracking/test/test_trackers.cpp

```diff
@@ -567,6 +567,38 @@ TEST_P(DistanceAndOverlap, Scaled_Data_CSRT)
   test.run();
 }
 
+TEST(GOTURN, memory_usage)
+{
+  cv::Rect2d roi(145, 70, 85, 85);
+  cv::Mat frame;
+
+  std::string model = cvtest::findDataFile("dnn/gsoc2016-goturn/goturn.prototxt");
+  std::string weights = cvtest::findDataFile("dnn/gsoc2016-goturn/goturn.caffemodel", false);
+
+  std::string dataPath = 
+    "%YAML:1.0\n"
+    "---\n"
+    "prototxt: " + model + "\n"
+    "caffemodel: " + weights + "\n";
+  cv::FileStorage goturnStorageRead( dataPath, cv::FileStorage::READ | cv::FileStorage::MEMORY);
+  cv::FileNode goturnNode = goturnStorageRead.root();
+
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create(TrackerGOTURN::Params());
+  string inputVideo = cvtest::findDataFile("tracking/david/data/david.webm");
+  cv::VideoCapture video(inputVideo);
+  tracker->read(goturnNode);
+
+  video >> frame;
+  tracker->init(frame, roi);
+  for (int nframes = 0; nframes < 15; ++nframes)
+  {
+    cout << "Frame: " << nframes << std::endl;
+    video >> frame;
+    if (frame.empty())
+        break;
+    tracker->update(frame, roi);
+  }
+}
```

Comment:
Please add sanity check for final bounding box position (compare values)

Reply:
Do you want just compare values ```[x, y, w, h]``` with the ground truth?

Reply:
Not sure what is ground truth in this case but yes - check bounding box location.

Reply:
The used video has .txt file with ground truth.